engine/xml-config/src/main/java/org/datacleaner/job/JaxbJobReader.java

```diff
@@ -533,13 +534,22 @@ private void wireOutputDataStreams(final Map<ComponentType, ComponentBuilder> co
 
                 for (int i = 0; i < sourceColumnTypes.size(); i++) {
                     final ColumnType sourceColumnPath = sourceColumnTypes.get(i);
-                    final String outputStreamColumnPathName =
-                            getOutputStreamColumnPath(sourceColumnTypes.get(i).getPath(), componentType,
-                                    componentBuilder, i);
+                    final List<MetaModelInputColumn> findSourceColumn = sourceColumns.stream().filter(
+                            sourceCol -> sourceCol.getName().equals(sourceColumnPath.getPath())).collect(Collectors
+                                    .toList());
+                    final String outputStreamColumnPathName;
+                    if (findSourceColumn.size() > 0) {
+                        outputStreamColumnPathName = getOutputStreamColumnPath(findSourceColumn.get(0).getName(),
+                                componentType, componentBuilder, i);
+                    } else {
+                        outputStreamColumnPathName = getOutputStreamColumnPath(sourceColumns.get(i).getName(),
+                                componentType, componentBuilder, i);
```

Comment:
Hmmm... OTOH, it's also missing from the completeness _inputs_ and I don't it this part of the code should affect that at all. Probably human error, or something else.